pep/app/pods/search/controller.ts

```diff
@@ -27,15 +38,32 @@ export default class Search extends Controller {
     //workaround for bug w/array-based query param values
     //@see https://github.com/emberjs/ember.js/issues/18981
     //@ts-ignore
-    queryParams = ['q', { _searchTerms: 'searchTerms' }, 'matchSynonyms', { _facets: 'facets' }];
+    queryParams = [
+        'q',
+        { _searchTerms: 'searchTerms' },
+        'matchSynonyms',
+        'citedCount',
+        'viewedCount',
+        'viewedPeriod',
+        { _facets: 'facets' }
+    ];
+
+    @tracked isLimitOpen: boolean = false;
     @tracked q: string = '';
     @tracked matchSynonyms: boolean = false;
+    @tracked citedCount: string = '';
+    @tracked viewedCount: string = '';
+    @tracked viewedPeriod: ViewPeriod = SEARCH_DEFAULT_VIEW_PERIOD;
     @tracked paginator!: Pagination<Document>;
 
     @tracked currentSmartSearchTerm: string = '';
     @tracked currentSearchTerms: SearchTermValue[] = [];
     @tracked currentMatchSynonyms: boolean = false;
+    @tracked currentCitedCount: string = '';
+    @tracked currentViewedCount: string = '';
+    @tracked currentViewedPeriod: ViewPeriod = SEARCH_DEFAULT_VIEW_PERIOD;
```

Comment:
note for review (and my future self), going to work on converting this to the new query params hook in the next search PR, as it'll likely require a decent amount of refactoring of all the search stuff, and can't hold this initial PR up any longer.